addons/website_sale_collect/static/src/js/checkout.js

```diff
@@ -4,6 +4,7 @@ import publicWidget from '@web/legacy/js/public/public_widget';
 publicWidget.registry.WebsiteSaleCheckout.include({
     events: Object.assign({}, publicWidget.registry.WebsiteSaleCheckout.prototype.events, {
         'click .js_delete_product': '_onClickDeleteProduct',
+        'click .o_wsale_reload': '_onClickReload',
```

Comment:
I think we can merge these two functions as they are both called on clicking 'remove' or 'save for later' buttons. And use only one class instead of two.
```suggestion
        'click .js_checkout_remove_product': '_onClickRemoveProduct',
```

Reply:
Well I could just have used the class js_delete_product which is present in website_sale_collect, but the issue is occuring due to reload. The following events happen when we add to wishlist. Ref [video](https://drive.google.com/file/d/1s6F0ErFFEN15ICafb3FxdAcog0fpn-ic/view?pli=1)

1. The product will be added in wishlist - 1
2. Due to reload, page will be reloaded hence wishlist - 0
3. Then again the wishlist - 1

Also, if you can refer our previous conversations, you might get a more clear picture of this situation. :smile:

Reply:
Yeah, from what I have understood - product is removed before reloading. 
Implemented your output and currently its working well! :smile:

Reply:
I am worried that if the RPC removing a product takes longer to process, reloading will not display changes because they will happen later than reloading.